mmv1/products/bigquery/DatasetAccess.yaml

```diff
@@ -33,7 +33,7 @@ docs:
     The API does accept both formats but it will always return the legacy format which results in Terraform
     showing permanent diff on each plan and apply operation.
 base_url: 'projects/{{project}}/datasets/{{dataset_id}}'
-self_link: 'projects/{{project}}/datasets/{{dataset_id}}'
+self_link: 'projects/{{project}}/datasets/{{dataset_id}}?accessPolicyVersion=3'
```

Comment:
The id of the resource has been changed as well, which is a breaking change and should not happen in a minor release.

This is the generated code in google-beta provider 
https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-12475-old..auto-pr-12475#diff-60e14eb86e5d66ff60e308eb2a862558de7a388d28410365ce544d1cb961a5e3R536